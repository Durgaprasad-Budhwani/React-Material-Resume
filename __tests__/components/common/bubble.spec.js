/**
 * Created by Durgaprasad Budhwani on 10/6/2017.
 */
/* global expect,it */
import React from 'react';
import Bubble from '../../../src/components/common/bubble';
import renderer from 'react-test-renderer';

it('renders bubble', () => {
    const tree = renderer.create(<Bubble content="Hello" />).toJSON();
    expect(tree).toMatchSnapshot();
});
