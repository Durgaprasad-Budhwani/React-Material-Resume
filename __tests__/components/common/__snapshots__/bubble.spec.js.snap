// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders bubble 1`] = `
<span
  className="Bubble-container-1"
>
  Hello
</span>
`;

exports[`renders bubble without crashing 1`] = `
<span
  className="Bubble-container-1"
>
  Hello
</span>
`;
